### PR TITLE
Fixes #3610 - explains error detection problems

### PR DESCRIPTION
**Pull request by ares**
see http://projects.theforeman.org/issues/3610 for more details

### CODE REVIEW

**Comment by ares** [comment]
@GregSutcliffe could you please review my English?

File: bin/foreman-installer [hunk 1]
```diff
@@ -19,7 +19,11 @@ def get_param(mod, name)
 end
 
 # functions specific to foreman installer
-Kafo::KafoConfigure.app_option '--reset-foreman-db', :flag, 'Drop foreman database first? You will lose all data!', :default => false
+Kafo::KafoConfigure.app_option '--reset-foreman-db', :flag,
+                               "Drop foreman database first? You will lose all data! Unfortunately we\n" +
+                               "can't detect a failure at the moment so you should verify the success\n" +
+                               'manually. E.g. dropping can fail when DB is currently used',
```
Comment (GregSutcliffe) [inline bin/foreman-installer]: Almost perfect - try "when DB is currently in use" :)

**Comment by ares** [comment]
Thanks :-) Feel free to merge then.

**Comment by domcleal** [comment]
Merged as 76518ec, thanks @ares.

**Pull request closed without merging**